ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDLaList.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.cmc.ccd.domain;
+
+public enum CCDLaList {
+
+    REFER_TO_JUDGE("OCMC boxwork"),
+    REFER_TO_COURT_STAFF("Any courtstaff with direction"),
```

Comment:
Court Staff needs to be 2 words

Reply:
Actually removing **REFER_TO_COURT_STAFF("Any courtstaff with direction"),** this is not relevant to this story.